Src/library/memory_analyzer/object_graph/ma_figure_factory.e

```diff
@@ -60,17 +62,19 @@ feature -- Basic operations
 --					end
 --				end
 --			end
+			to_implement (generator + ".model_from_xml: Not yet implemented!")
+			;(create {EXCEPTIONS}).die (-1)
```

Comment:
In this case, I am not sure .. this is not implemented, so a "die" sounds good to me.